src/hibernate-resume/hibernate-resume.c

```diff
@@ -32,6 +33,7 @@ static int help(void) {
                "\n%sInitiate resume from hibernation.%s\n\n"
                "  -h --help            Show this help\n"
                "     --version         Show package version\n"
+               "     --clear-efi       Clear stale HibernateLocation EFI variable and exit\n"
```

Comment:
Well, the "and exit" is redundant. Of course we exit after executing the command.

And I'd drop the reference to the name of the EFI variable. That smells a bit too much of an implementation detail. i.e. sounds great for the man page, but I would not stuff it into the --help text.

And this doesn't just clear stale info, it clears any info afaics, stale or not. Hence, don#t say "stale". I mean, it's hopefully typically used to drop stale info, but that's not really strictly the only case.

Hence maybe "Clear hibernation stroage information from EFI variables" or so?

Reply:
> Well, the "and exit" is redundant. Of course we exit after executing the command.

Hmm, I see an ambiguity here: people might assume that if and only if this option is specified, the efivar gets cleared before resuming. But it actually just clears it and quits.